circle.yml

```diff
@@ -28,7 +28,7 @@ mainBuildFilters: &mainBuildFilters
     branches:
       only:
         - develop
-        - 10.0-release
+        - chore/beta-release-m1-support
         - fix-darwin-win32-node-modules-install
```

Comment:
@JessicaSachs You'll want to leave the 10.0-release branch since this would erase those rules when it merges into 10.0-release. You'll want to replace anywhere that `fix-darwin-win32-node-modules-install` is written with this branch: `chore/beta-release-m1-support`

Reply:
I'm reverting this change completely after i test.